wix/Build.OpenJDK_generic.cmd

```diff
@@ -71,30 +71,26 @@ REM
 
 REM Cultures: https://msdn.microsoft.com/de-de/library/ee825488(v=cs.20).aspx
 SET PRODUCT_SKU=OpenJDK
-SET PRODUCT_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%.%PRODUCT_BUILD_NUMBER%
+SET PRODUCT_FULL_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%.%PRODUCT_BUILD_NUMBER%
 SET ICEDTEAWEB_DIR=.\SourceDir\icedtea-web-image
 
 SETLOCAL ENABLEDELAYEDEXPANSION
-REM JEP322 alike
-SET JEP322_BASE_NAME="unknown"
-SET PRODUCT_FULL_VERSION=%PRODUCT_MAJOR_VERSION%u%PRODUCT_MAINTENANCE_VERSION%-b%PRODUCT_BUILD_NUMBER%
+SET PRODUCT_SHORT_VERSION=%PRODUCT_MAJOR_VERSION%u%PRODUCT_MAINTENANCE_VERSION%-b%PRODUCT_BUILD_NUMBER%
 IF %PRODUCT_CATEGORY% EQU jre SET JRE=-jre
 IF %PRODUCT_MAJOR_VERSION% GEQ 10 (
-    IF %PRODUCT_MAJOR_VERSION% EQU 11 SET LTS_VERSION=-LTS
```

Comment:
no more -LTS ?

Reply:
The only place the LTS_VERSION variable was used was in JEP322_NAME, which seemed to be used precisely nowhere.